cpp/src/io/utilities/column_buffer.cpp

```diff
@@ -33,7 +33,7 @@
 
 namespace cudf::io::detail {
 
-void gather_column_buffer::allocate_strings_data(rmm::cuda_stream_view stream)
+void gather_column_buffer::allocate_strings_data(rmm::cuda_stream_view stream, bool memset_data)
```

Comment:
Not being used, is it?

```suggestion
void gather_column_buffer::allocate_strings_data(rmm::cuda_stream_view stream)
```